deployments/data-hub/data-hub-configs.yaml

```diff
@@ -55,6 +55,32 @@ data:
             bucketName: '{ENV}-elife-data-pipeline'
             objectName: 'airflow-config/europepmc/biorxiv-medrxiv/{ENV}-state.json'
         batchSize: 1000
+  europepmc-labslink.config.yaml: |-
+    europePmcLabsLink:
+    - dataPipelineId: europepmc_labslink_data_pipeline_1
+      source:
+        bigQuery:
+          projectName: 'elife-data-pipeline'
+          sqlQuery: |-
+            SELECT DISTINCT article_doi
+            FROM `elife-data-pipeline.de_proto.v_sciety_event` AS event
+            WHERE event_name = 'EvaluationRecorded'
+                AND article_doi IS NOT NULL
+      xml:
+        providerId: '2112'
+        linkTitle: 'Read the evaluations of this preprint'
+        linkPrefix: 'https://sciety.org/articles/activity/'
+      target:
+        ftp:
+          hostname: 'labslink.ebi.ac.uk'
```

Comment:
The host should point to an internal host for staging.
How do we make it use a different hostname between staging and prod?
Do we need to maintain two "europepmc-labslink.config.yaml" config map entries?